src/track/beatfactory.cpp

```diff
@@ -92,29 +94,32 @@ QString BeatFactory::getPreferredSubVersion(
 mixxx::BeatsPointer BeatFactory::makePreferredBeats(
         const QVector<double>& beats,
         const QHash<QString, QString>& extraVersionInfo,
-        const bool bEnableFixedTempoCorrection,
-        const bool bEnableOffsetCorrection,
-        const int iSampleRate,
-        const int iTotalSamples,
-        const int iMinBpm,
-        const int iMaxBpm) {
+        bool bEnableFixedTempoCorrection,
+        bool bEnableOffsetCorrection,
+        mixxx::audio::SampleRate sampleRate,
+        SINT totalSamples,
```

Comment:
Are these mono samples or stereo samples (i.e. frames)? If the latter, let's use `totalFrames` instead.

Reply:
It are frames * 2 in this case.

Reply:
Ok, than let's leave it as-is.